apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/CharSequenceExpectationSamples.kt

```diff
@@ -186,4 +186,22 @@ class CharSequenceExpectationSamples {
             expect("ABC").notToMatch("A(B)?C".toRegex())
         }
     }
+
+    @Test
+    fun lengthFeature() {
+        expect("Hello").length.toBeGreaterThan(3)
+
+        fails {
+            expect("Hi").length.toBeGreaterThan(3)
+        }
+    }
+
+    @Test
+    fun length() {
+        expect("Hello").length { toBeGreaterThan(3) }
+
+        fails {
+            expect("Hi").length { toBeGreaterThan(3) }
+        }
+    }
```

Comment:
noted it wrong in the issue description (fixed it now). Please move those two to a new file CharSequencenFeatureExtractorSamples.kt